doc/project/webassembly.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: doc
+title: Emitting JavaScript modules
+---
+
+# Experimental WebAssembly backend
+
+Since Scala.js 1.17.0, there is an *experimental* WebAssembly backend (Wasm for short).
+Under some conditions, it is designed to be a *drop-in* replacement for the usual JavaScript backend.
+
+## Experimental status
+
+Being experimental means that:
+
+* The Wasm backend may be removed in a future *minor* version of Scala.js (or moved to a separate plugin).
+* Future versions of Scala.js may emit Wasm that requires *newer* versions of Wasm engines, dropping support for older engines.
+
+However, we do *not* expect the *quality* of the Wasm backend to be any lower than the JS backend.
+Feel free to report any issue you may experience with the same expectations as for the JS backend.
```

Comment:
Just thought about this: You might want to drop a line here about semantics v.s. performance:
IIUC as far as semantics are concerned, we expect full parity, as far as performance is concerned, we still expect improvements. (IMHO the text in the release announcement captures this quite nicely, probably can just be copied).

Reply:
Also codesize and linker incrementality, actually?